client/src/pages/LogInForm.js

```diff
@@ -0,0 +1,90 @@
+/* eslint-disable no-trailing-spaces */
+/* eslint-disable no-unused-vars */
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+
+function LogInForm(){
+    const [emailInput , setEmailInput]=useState("");
+    const [passwordInput , setPasswordInput]=useState("") ;
+    const [emailError , setEmailError]=useState("") ;
+    const [passwordError , setPasswordError]=useState("");
+    
+
+    //record changes for email address input
+    const handleEmailChange = (e) => {
+			const email = e.target.value; 
+			setEmailInput(email); 
+		};
+
+    //record changes on password input
+    const handlePasswordChange=(e)=>{
```

Comment:
There are a lot of indentation and formatting errors in this PR. Please fix them